api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/TxNewDataset.java

```diff
@@ -0,0 +1,206 @@
+		            patientWithHistoricalDrugStartDateObs, transferredFromOtherHealthFacility, PatientBetween1And9Years);
+		CohortIndicator patientBetween1And9YearsEnrolledInHIVStartedARTIndicator = GenericIndicators.newCohortIndicator(
+		    "patientBelow1YearEnrolledInHIVStartedARTIndicator", patientBetween1And9YearsEnrolledInHIVStartedART,
+		    ParameterizableUtil
+		            .createParameterMappings("onOrAfter=${startDate},onOrBefore=${endDate},effectiveDate=${endDate}"));
+		dataSetDefinition.addColumn(
+		    "119",
+		    "TX_NEW: New on ART: Patients Between 1 and 9 years",
+		    new Mapped(patientBetween1And9YearsEnrolledInHIVStartedARTIndicator, ParameterizableUtil
+		            .createParameterMappings("startDate=${startDate},endDate=${endDate}")), "");
+		
+		// Male		
+		int i = 2;
+		for (AgeCohortDefinition ageCohort : agesRange) {
+			CompositionCohortDefinition patientInYearRangeEnrolledInARTStarted = compositionCohortQueries
+			        .getPatientInYearRangeEnrolledInARTStarted(inARTProgramDuringTimePeriod, patientWithSTARTDRUGSObs,
+			            patientWithHistoricalDrugStartDateObs, transferredFromOtherHealthFacility, PatientBetween1And9Years,
+			            ageCohort, males);
+			CohortIndicator patientInYearRangeEnrolledInHIVStartedARTIndicator = GenericIndicators.newCohortIndicator(
+			    "patientInYearRangeEnrolledInHIVStartedARTIndicator", patientInYearRangeEnrolledInARTStarted,
+			    ParameterizableUtil
+			            .createParameterMappings("onOrAfter=${startDate},onOrBefore=${endDate},effectiveDate=${endDate}"));
+			dataSetDefinition.addColumn(
+			    "1M" + i,
+			    "Males:TX_NEW: New on ART by age and sex: " + ageCohort.getName(),
+			    new Mapped(patientInYearRangeEnrolledInHIVStartedARTIndicator, ParameterizableUtil
+			            .createParameterMappings("startDate=${startDate},endDate=${endDate}")), "");
+			
+			i++;
+		}
+		
+		// Female		
+		int j = 2;
+		for (AgeCohortDefinition ageCohort : agesRange) {
+			CompositionCohortDefinition patientInYearRangeEnrolledInARTStarted = compositionCohortQueries
+			        .getPatientInYearRangeEnrolledInARTStarted(inARTProgramDuringTimePeriod, patientWithSTARTDRUGSObs,
+			            patientWithHistoricalDrugStartDateObs, transferredFromOtherHealthFacility, PatientBetween1And9Years,
+			            ageCohort, females);
+			CohortIndicator patientInYearRangeEnrolledInHIVStartedARTIndicator = GenericIndicators.newCohortIndicator(
+			    "patientInYearRangeEnrolledInHIVStartedARTIndicator", patientInYearRangeEnrolledInARTStarted,
+			    ParameterizableUtil
+			            .createParameterMappings("onOrAfter=${startDate},onOrBefore=${endDate},effectiveDate=${endDate}"));
+			dataSetDefinition.addColumn(
+			    "1F" + j,
+			    "Females:TX_NEW: New on ART by age and sex: " + ageCohort.getName(),
+			    new Mapped(patientInYearRangeEnrolledInHIVStartedARTIndicator, ParameterizableUtil
```

Comment:
@rubailly these lines are causing unchecked type safety warnings to be thrown. Please resolve these

Reply:
[Done](https://github.com/esaude/openmrs-module-eptsreports/commit/0b97c79036b048b856fcb880cca769d40b1654c3)